main.js

```diff
@@ -564,6 +571,23 @@ app.on('ready', function() {
       require('open')('mailto:project@openbazaar.org?subject=OpenBazaar Debug Report&body=' + body);
 
     }},
+    {label: 'View Python Error Log', type: 'normal', click: function() {
```

Comment:
I'll take that back as it doesn't seem to flag anything now when running it. I'm not sure why it was giving me issues about other files before.